packages/benchmark-test/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+// This file can contain .js-specific Typescript compiler config.
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {},
+  "include": [
+    "tools",
+    "test",
+    "rollup.config.js"
```

Comment:
I did not move "rollup.config.js" to a subdirectory, because it is directly used by "yarn test" at the top.